jupyterlab/labbox_ephys_widgets_jp/css/styles.css

```diff
@@ -0,0 +1,44 @@
+* {
```

Comment:
Styles file so that jupyter styling matches more closely to the web gui. Eventually the two should refer to the same styles.css in the extensions/ directory.